backend-stub/index.js

```diff
@@ -30,11 +30,13 @@ const apiCall = async (url, data) => {
 
 // When a user clicks submit, the application creates a post request to the endpoint
 app.post("/api/feedback", (req, res) => {
```

Comment:
Do we still need this?

Reply:
Probably not— I don't think this file does anything currently... might want to keep it though in case we want to test things locally before creating an Azure function for it?

Reply:
Hmm ok. So maybe keep file but delete this api calls?